Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/DownloadResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Frends.AmazonS3.DownloadObject
```

Comment:
Please use one-line namespace definition. Also, the namespace should be `Frends.AmazonS3.DownloadObject.Definitions`.